configs/webserver/halfstack.conf

```diff
@@ -33,6 +33,8 @@ max_file_upload_size = 4294967296
 
 [pipeline]
 endpoint = "http://127.0.0.1:9500"
+frontend-endpoint = "http://127.0.0.1:3000"
+hide-button = false
```

Comment:
`hide-button` is a general name. Let's make it more specific. @rapsealk

Reply:
@yomybaby How about `hide-shortcut-button`? It would be `pipeline.hide-shortcut-button` on server side.

Reply:
`hide-side-menu-button`? ;)